TVRemoteControl.xcodeproj/project.pbxproj

```diff
@@ -41,10 +89,118 @@
 			isa = PBXGroup;
 			children = (
 				1922112B21AFC39F00C0A68C /* Rswift.framework */,
+				2B1AC8592220FC88009BBBF1 /* MMLanScan */,
```

Comment:
How I can see that's not Framework. I think better to move into a separated directory in the project. For instance, `TVRemoteControl/Third-Party/MMLanScan`

Reply:
Moved to app folder.